raiden_contracts/tests/test_contracts_compilation.py

```diff
@@ -232,3 +226,16 @@ def test_contract_manager_json(tmpdir):
     ContractManager(contracts_source_path()).compile_contracts(precompiled_path)
     # try to load contracts from a precompiled file
     contract_manager_meta(precompiled_path)
+
+
+def test_contract_manager_constructor_does_not_invent_version():
+    """ ContractManager should not invent a version string """
+    manager = ContractManager(contracts_precompiled_path(version=None))
+    assert manager.contracts_version is None
+
+
+@pytest.mark.parametrize('version', [CONTRACTS_VERSION, '0.3._', '0.4.0'])
```

Comment:
Could you add `0.9.0` to the list? just to make sure it also works

Reply:
Will do in this PR.